scripts/release_helper.sh

```diff
@@ -0,0 +1,601 @@
+#!/bin/bash
+
+# Get the path of the script
+SCRIPT_PATH=$(dirname $(realpath -s $0))
```

Comment:
```suggestion
SCRIPT_PATH=$(dirname "$(realpath -s "$0")")
```

Reply:
Without this, a space in a directory name will result in the wrong `SCRIPT_PATH`.